docs/pages/guides/overview.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Overview
+hideTOC: true
+---
+
+This section contains information about the following process and services:
+
+- **Development process**: This section contains guides about additional configurations and workflows that you may require during the development process to help you develop, deploy, and maintain your app. It contains in-depth information about [app config](/workflow/configuration/), [permissions](/guides/permissions/), [universal links](/guides/deep-linking/), [custom native code](/workflow/continuous-native-generation/), [web](/workflow/web/), and more.
+- **Expo Application Services (EAS)**: Deeply integrated cloud services for Expo and React Native apps, from the team behind Expo. There are different services available such as [EAS Build](/build/introduction/), [EAS Submit](/submit/introduction/), [EAS Update](/eas-update/introduction/), [EAS Metadata](/eas/metadata/) and [EAS Insights](/eas-insights/introduction/).
```

Comment:
```suggestion
- **Expo Application Services (EAS)**: Deeply integrated cloud services for Expo and React Native apps, from the team behind Expo. Learn about [EAS Build](/build/introduction/), [EAS Submit](/submit/introduction/), [EAS Update](/eas-update/introduction/), [EAS Metadata](/eas/metadata/), and [EAS Insights](/eas-insights/introduction/).
```

Adding the oxford comma too, since that's in our styleguide 🤓